test/specs/rules/require-use-strict.js

```diff
@@ -0,0 +1,93 @@
+var Checker = require('../../../lib/checker');
+var expect = require('chai').expect;
+
+var funcX = 'function x() {"use strict";\nreturn true;}';
+var funcY = 'function x() {return true;}';
+
+function makeChecker(rules) {
```

Comment:
Sounds like it would easier to do it like [this](https://github.com/jscs-dev/node-jscs/blob/fbe9bf4ad5ab3e2645affee07a5fc5ff19cea35f/test/specs/rules/disallow-arrow-functions.js#L7)